sdk/program/src/rent.rs

```diff
@@ -96,6 +93,33 @@ impl Rent {
     }
 }
 
+/// Enumerate return values from `Rent::due()`
+#[derive(Debug, Copy, Clone, Eq, PartialEq)]
+pub enum RentDue {
+    /// Used to indicate the account is rent exempt
+    Exempt,
+    /// The account owes rent, and the amount is the field
+    Paying(u64),
+}
```

Comment:
Here's the `RentDue` enum. Now the return value from `Rent::due()` is more clear. (I.e. when exempt, no need to also supply an amount, since that's not meaningful). Very much based on `Option`.

Reply:
This is great!